redwood-layout-uiview/src/commonMain/kotlin/app/cash/redwood/layout/uiview/UIViewBox.kt

```diff
@@ -103,10 +112,20 @@ internal class UIViewBox : Box<UIView> {
             is VerticalAlignment -> {
               itemVerticalAlignment = childModifier.alignment
             }
+            is Width -> {
+              requestedWidth = childModifier.width.toPlatformDp()
+            }
+            is Height -> {
+              requestedHeight = childModifier.height.toPlatformDp()
+            }
           }
         }
 
-        // Compute origin and stretch if needed
+        // Use requested modifiers, otherwise use the size established from sizeToFit().
+        var childWidth: CGFloat = requestedWidth?.let { it } ?: view.frame.useContents { this.size.width }
+        var childHeight: CGFloat = requestedHeight?.let { it } ?: view.frame.useContents { this.size.height }
```

Comment:
`?.let { it }` is an identity transformation
```suggestion
        var childWidth: CGFloat = requestedWidth ?: view.frame.useContents { this.size.width }
        var childHeight: CGFloat = requestedHeight ?: view.frame.useContents { this.size.height }
```